examples/capacitor-angular/README.md

```
@@ -0,0 +1,166 @@
+  navigator,
+  onNavigatorWarning: (warning) => {
+    console.warn('[NavigatorWarning]', warning);
+  },
+  isNativeApp: true,
+  postLoginRedirectUrl: 'myapp://callback',
+});
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)